src/Modules/CalcSections.lua

```diff
@@ -951,105 +951,149 @@ return {
 	}, },
-	{ label = "Shock Effect Mod", flag = "shock", { format = "+{0:output:ShockEffectModDisplay}%",
-		{ label = "Player modifiers", modName = "EnemyShockEffect", cfg = "skill" },
+	{ label = "Maximum Brittle", bgCol = colorCodes.BRITTLEBG, flag = "brittle", { format = "{0:output:MaximumBrittle}%",
+		{ modName = "BrittleMax" },
+	}, },
+	{ label = "Current Brittle", bgCol = colorCodes.BRITTLEBG, haveOutput = "CurrentBrittle", { format = "{0:output:CurrentBrittle}%",
+		{ label = "Configured Brittle", modName = "BrittleVal", enemy = true, modType = "BASE" },
+		{ label = "Guaranteed Brittles", modName = "BrittleOverride", modType = "BASE" },
+	}, },
+	{ label = "Shock Effect Mod", bgCol = colorCodes.SHOCKBG, flag = "shock", { format = "x {2:output:ShockEffectMod}",
+		{ breakdown = "ShockEffectMod" },
+		{ breakdown = "MainHand.ShockDPS" },
+		{ breakdown = "OffHand.ShockDPS" },
+		{ breakdown = "ShockDPS" },
+		{ label = "Player modifiers", modName = { "EnemyShockEffect", "ShockAsThoughDealing" }, cfg = "skill" },
 		{ label = "Enemy modifiers", modName = "SelfShockEffect", enemy = true },
 	}, },
-	{ label = "Chance to Shock", flag = "shock", { format = "{0:output:ShockChance}%", 
+	{ label = "Chance to Shock", bgCol = colorCodes.SHOCKBG, flag = "shock", { format = "{0:output:ShockChance}%",
 		{ breakdown = "MainHand.ShockChance" },
 		{ breakdown = "OffHand.ShockChance" },
 		{ breakdown = "ShockChance" },
 		{ label = "Player modifiers", modName = "EnemyShockChance", cfg = "skill" },
 		{ label = "Enemy modifiers", modName = "SelfShockChance", enemy = true },
 	}, },
-	{ label = "Shock Dur. Mod", flag = "shock", { format = "x {2:output:ShockDurationMod}",
-		{ breakdown = "ShockDurationMod" },
-		{ breakdown = "MainHand.ShockDPS" },
-		{ breakdown = "OffHand.ShockDPS" },
-		{ breakdown = "ShockDPS" },
-		{ label = "Player modifiers", modName = { "EnemyShockDuration", "ShockAsThoughDealing" }, cfg = "skill" }, 
-		{ label = "Enemy modifiers", modName = "SelfShockDuration", enemy = true }, 
+	{ label = "Shock Duration", bgCol = colorCodes.SHOCKBG, flag = "shock", { format = "{2:output:ShockDuration}s",
+		{ breakdown = "ShockDuration" },
+		{ label = "Player modifiers", modName = "EnemyShockDuration", cfg = "skill" },
+		{ label = "Enemy modifiers", modName = { "SelfShockDuration", "BuffExpireFaster" }, enemy = true },
 	}, },
-	{ label = "Maximum Shock", flag = "shock", { format = "{0:output:MaximumShock}%",
-		{ modName = "ShockMax" }, 
+	{ label = "Maximum Shock", bgCol = colorCodes.SHOCKBG, flag = "shock", { format = "{0:output:MaximumShock}%",
+		{ modName = "ShockMax" },
 	}, },
-	{ label = "Current Shock", haveOutput = "CurrentShock", { format = "{0:output:CurrentShock}%",
-		{ label = "Configured Shock", modName = "ShockVal", enemy = true, modType = "BASE" }, 
+	{ label = "Current Shock", bgCol = colorCodes.SHOCKBG, haveOutput = "CurrentShock", { format = "{0:output:CurrentShock}%",
```

Comment:
This line still shows up if you take the Shaper of Storms ascendancy and have any source of "CannotShock" (Secrets of Suffering, in this case):

![image](https://user-images.githubusercontent.com/1209372/151511632-b4dbab02-5c97-4aed-b1f8-05b25b76acd2.png)